rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -216,7 +216,11 @@ class Publisher : public PublisherBase
   virtual void
   publish(std::unique_ptr<MessageT, MessageDeleter> & msg)
   {
-    this->do_inter_process_publish(msg.get());
+    bool inter_process_subscriptions_exist =
```

Comment:
Similar to the above I would suggest naming this one `has_inter_process_subscriptions`.

Reply:
I don't find this name any better or worse.